core/src/test/scala/org/apache/spark/io/ZStandardBenchmark.scala

```diff
@@ -49,6 +49,7 @@ object ZStandardBenchmark extends BenchmarkBase {
       val benchmark2 = new Benchmark(name, N, output = output)
       decompressionBenchmark(benchmark2, N)
       benchmark2.run()
+      parallelCompressionBenchmark()
```

Comment:
The attached benchmark result didn't have this, does it?

Reply:
Ya, I will update them after CI jobs finished